client/src/components/RadioInput.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
```

Comment:
```suggestion
  justify-content: left;
```
Overall good development 👍 

The only thing is that in your preview in storybook, the radiobuttons and labels do not seem to be aligned. I don't know the overall context but I added a suggestion to align your labels to the left.

Reply:
Thanks for your tip. It worked out great with `justify-content: left` 👌🏼